webviz_config/deprecation_decorators.py

```diff
@@ -8,10 +8,20 @@
 
 def deprecated_plugin(
     short_message: str = "", long_message: str = ""
-) -> Callable[[WebvizPluginABC], WebvizPluginABC]:
+) -> Callable[[Any], Any]:
```

Comment:
```suggestion
) -> Callable[Type[WebvizPluginABC], Type[WebvizPluginABC]]:
```
Would this work instead? (reference: https://mypy.readthedocs.io/en/latest/kinds_of_types.html#the-type-of-class-objects)

Reply:
Det har oppført seg litt rart i VS Code hos meg. Men når jeg kjører `mypy` direkte virker det som feilmeldingene var borte :)